library/src/enabled/java/com/telefonica/tweaks/Tweaks.kt

```diff
@@ -39,6 +41,12 @@ open class Tweaks {
 
     open fun <T> getTweakValue(key: String): Flow<T?> = tweaksBusinessLogic.getValue(key)
 
+    open fun <T> getTweakValue(key: String, defaultValue: T): Flow<T> = getTweakValue<T>(key).map { it ?: defaultValue }
```

Comment:
i think it should return a Flow of nullable `T` (`T?`). `defaultValue` should be nullable as well

Reply:
well, after rethink it, if you want a null default value you would probably be using the existing method. So ignore my comment 😄

Reply:
Exactly, the idea is to avoid having a nullable result